content/docs/account/billing.md

```diff
@@ -23,25 +23,25 @@ We won't automatically stop your instances and risk interrupting your services.
 
 ## Billing for shut down resources
 
-If an instance or Kubernetes cluster is shut down, it still takes up resources, so a stopped instance or cluster is chargeable is exactly the same as an active instance or cluster. In other words, The hourly price will be added to your monthly charges while the instances are in your account, running or not.
+If an instance or Kubernetes cluster is shut down, it still takes up resources, so a stopped instance or cluster is chargeable, and it is exactly the same as an active instance or cluster. In other words, The hourly price will be added to your monthly charges while the instances are in your account, running or not.
```

Comment:
Thanks for spotting this. I believe the best way to state this would be "...cluster is chargeable, and is exactly teh same as an active instance or cluster."

Reply:
Noted.